pact.cabal

```diff
@@ -80,6 +80,7 @@ library
     -Wincomplete-record-updates
     -Wincomplete-uni-patterns
     -Wredundant-constraints
+    -Wno-missed-extra-shared-lib
```

Comment:
Quoting Austin in his PR to chainweb-node:

> When running Template Haskell, GHC sees that some code it loads has dependent modules that also have a dependency on the zlib shared object via an extra-libraries clause; GHC can't find or load this module, issuing a warning, but it doesn't matter because the Template Haskell code doesn't actually use zlib despite having a transitive dependency on it.

> Silence this warning, as it's extremely annoying to see while working on any files that use Template Haskell for any purposes e.g. deriving lenses.

I have been getting this warning on chainweb-node and pact for a while now.